src/index.html

```diff
@@ -6,9 +6,91 @@
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header>
+      <a 
+        href="#home"
+        class="logo"
+      >
+        <img
+          src="/images/logo.png" 
+          alt="#"
+        >
+      </a>
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a
+              href="#apple"
+              class="link is-active"
```

Comment:
```suggestion
              class="nav__link is-active"
```

You also need to add the correct class for links due to [BEM](https://mate-academy.github.io/style-guides/htmlcss.html#bem-element-nesting)